manifests/v1beta1/installs/katib-openshift/patch-pvc-unset-storageClass.yaml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
Please, can you move all patches under `katib-openshift/patches` like for other installs: https://github.com/kubeflow/katib/tree/83678f1c3dad9bf648c7c0fb74a6ebd7323c3d3f/manifests/v1beta1/installs/katib-with-kubeflow or https://github.com/kubeflow/katib/tree/83678f1c3dad9bf648c7c0fb74a6ebd7323c3d3f/manifests/v1beta1/installs/katib-cert-manager